services/shhext/service.go

```diff
@@ -214,6 +214,8 @@ func (s *Service) retrieveMessagesLoop(tick time.Duration, cancel <-chan struct{
 						}
 						dedupMessage.Message.Payload = message.DecryptedPayload
 						dedupMessage.Payload = string(message.DecryptedPayload)
+						dedupMessage.ParsedMessage = message.ParsedMessage
```

Comment:
I believe it's known and it will be optimized later but the payload is tripled:
- `dedupMessage.Message.Payload`
- `dedupMessage.Payload`
- `dedupMessage.ParsedMessage` in decoded form

Reply:
Yes, I think we can remove now Message.Payload, but I will double check